test/delayed_queue_test.rb

```diff
@@ -109,14 +109,28 @@
 
   test 'handle_delayed_items with items in the future' do
     t = Time.now + 60 # in the future
-    Resque.enqueue_at(t, SomeIvarJob)
-    Resque.enqueue_at(t, SomeIvarJob)
 
-    # 2 SomeIvarJob jobs should be created in the "ivar" queue
-    Resque::Job.expects(:create).twice.with('ivar', SomeIvarJob, nil)
+    assert_equal 10, Resque::Scheduler.dequeue_batch_size
```

Comment:
Does this test care about the `dequeue_batch_size`?

Reply:
Good point. This is bad expression of my idea for this test. 

The intention is to test `handle_delayed_items` when there are more jobs in the queue than the batch size. I'll make it more clear by using a `batch_size` variable and doing `(batch_size * 3).times { Resque.enqueue_at(t, SomeIvarJob) }`